src/sql/gazetteer_geometry_views.sql

```diff
@@ -25,16 +25,23 @@ from
 
 CREATE OR REPLACE RULE feature_ref_point_ins AS ON INSERT TO feature_ref_point
 DO INSTEAD
-   (
-   INSERT INTO feature (feat_type, status, description, ref_point )
-   VALUES (NEW.feat_type, 'CURR', '', NEW.ref_point );
-   INSERT INTO name (feat_id, name, status )
-   VALUES (lastval(), NEW.name, 'UNEW' )
-   RETURNING
-     feat_id,
-     gaz_preferredName(feat_id) as name,
-     (SELECT f.feat_type FROM feature f WHERE f.feat_id = feat_id),
-     (SELECT f.ref_point FROM feature f WHERE f.feat_id  = feat_id));
+    (
+    INSERT INTO feature (feat_type, status, description, ref_point )
+    VALUES (NEW.feat_type, 'CURR', '', NEW.ref_point );
+    INSERT INTO name (feat_id, name, status )
+    VALUES (lastval(), NEW.name, 'UNEW' )
+    RETURNING
+        feat_id,
+        gaz_preferredname(feat_id),
+       	(SELECT feat_type
```

Comment:
It looks to me you can replace this subselect with  just `NEW.feat_type` here ?

Reply:
:worried:  `HINT:  There is an entry for table "*NEW*", but it cannot be referenced from this part of the query.`